For Beginners/200.md

```diff
@@ -0,0 +1,22 @@
+# Try Again
+
+If you take a look at various commands’ description, you will notice that a vast majority of them share the [same arguments](appendices/common-arguments.md). Four of these arguments are targeted at error handling: `errorcall`, `errorjump`, `errormessage` and `errorresult`. You can use one of them to convince a user to provide a correct input.
+
+You will use the `try` command, which is another block command: it tries to execute everything inside the block between `try` and `end` keywords. In case an error occurs within this block, the robot will do what one of the `error...` arguments instructs: call a procedure, jump to a label, display a message or just store the error information in a variable.
+
+Try this out:
+
+```G1ANT
+try errorcall askUser
+  call askUser
+  ♥input = ⟦integer⟧♥input
+end
```

Comment:
It's a good example of `try/errorcall`, but if you look at it deeper it won't guarantee that `input` will contain a number (type a letter twice) and therefore it is misleading and not a good code example.

Reply:
corrected